settings/package.json

```diff
@@ -13,7 +13,8 @@
 		"packages-update": "wp-scripts packages-update",
 		"start": "wp-scripts start --webpack-copy-php",
 		"test:unit": "wp-scripts test-unit-js",
-		"test:unit:watch": "wp-scripts test-unit-js --watch"
+		"test:unit:watch": "wp-scripts test-unit-js --watch",
+		"prepare": "cd .. && husky install settings/.husky"
```

Comment:
Do we need to add a step in the `readme` that asks folks to run this, or does it get run automatically?

Reply:
It would run automatically when `npm install` inside `settings`.

Reply:
Nice 👍🏻